auto_backup/data/backup_data.yml

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
```

Comment:
In this case we're talking maybe a ten minute conversion too, so really not too much skin off my back even if I didn't already want to see YML die in a fire 😄 

Just to note though, this isn't a test - it's the cron and message data.